x/ecocredit/migrations/v3/patch_test.go

```diff
@@ -535,6 +547,384 @@ func TestRedwoodMigrations(t *testing.T) {
 	})
 	require.NoError(t, err)
 
+	startDate, _ = time.Parse(time.RFC3339, "2018-05-07T18:07:46Z")
+	endDate, _ = time.Parse(time.RFC3339, "2024-06-07T18:07:53Z")
```

Comment:
A bit concerning that all these start and end dates do not have a 00:00:00 UTC time. We've discussed the possibility of enforcing this but not something we need to change in this pull request.